app/js/components/cohorts/cohortModal.js

```diff
@@ -68,13 +69,22 @@ class ActionsComponent extends Component {
                             name :'', 
                             description: ''
                     }));
+                    
                     $('#myCohort').modal('hide');          
                 });
         } else {
             this.setState({ error: "all fields are required" });
         }
     }
 
+    onSave(event) {
+        event.preventDefault();
+        const {name, description} = this.state;
+        if(name && description) {
```

Comment:
@dkayiwa An error message pops up to show that all fields are required. The data the user has input is not reset, to allow him to fill in the missing field. But if both the name and description are provided and the form data is saved then the fields are reset/cleared so that they are ready for the next set of data from the user.